lib/lighthouse/auth/client_credentials/service.rb

```diff
@@ -51,6 +51,7 @@ def build_assertion
       # @return [Hash] body of request to get access token
       #
       def build_request_body(assertion, scopes, auth_params = {})
+        auth_params = auth_params.nil? ? {} : auth_params
```

Comment:
This can be simplified to `auth_params = {} if auth_params.nil?`, assuming we need to use the `nil` guard